moodlecli/moodle.py

```diff
@@ -274,3 +275,10 @@ def get_quiz_attempt_details(self, attempt_id):
             "attemptid": attempt_id
         }
         return self._get(MOODLE_FUNC_GET_QUIZ_ATTEMPT, data)
+
+    def get_policy_acceptance_data(self, policyversionid, user_ids=[]):
```

Comment:
The way this is implemented is inconsistent in that you're leaking the Moodle convention all the way to the caller. Comparing to `get_user_uuids()` where we expect `user_ids` to be a list of ints and then do the conversion for the list of objects that Moodle would expect in the client method. To be consistent, I think this should be invocable as:

```python
 client.get_policy_acceptance_data(1, [1, 2, 3])
```